api/lib/application/sessions/session-controller.js

```diff
@@ -182,32 +181,31 @@ const getJuryCertificationSummaries = async function (
 
 const generateSessionResultsDownloadLink = async function (request, h, dependencies = { sessionResultsLinkService }) {
   const sessionId = request.params.id;
-  const sessionResultsLink = dependencies.sessionResultsLinkService.generateResultsLink(sessionId);
+  const i18n = request.i18n;
+  const sessionResultsLink = dependencies.sessionResultsLinkService.generateResultsLink({ sessionId, i18n });
 
   return h.response({ sessionResultsLink });
 };
 
 const getSessionResultsToDownload = async function (
   request,
   h,
-  dependencies = { tokenService, certificationResultUtils }
+  dependencies = { tokenService, getSessionCertificationResultsCsv }
 ) {
   const token = request.params.token;
   const { sessionId } = dependencies.tokenService.extractSessionId(token);
   const { session, certificationResults } = await usecases.getSessionResults({ sessionId });
 
-  const csvResult = await dependencies.certificationResultUtils.getSessionCertificationResultsCsv({
+  const csvResult = await dependencies.getSessionCertificationResultsCsv({
     session,
     certificationResults,
+    i18n: request.i18n,
```

Comment:
Tu créer une constante plus haut, on peut l'utiliser ici

```suggestion
    i18n,
```

Reply:
Parles-tu de la constante ligne 184 ? Car ce n'est pas la même fonction si oui.
Mais en revanche je peux harmoniser entre la fonction `getSessionResultsToDownload` et `generateSessionResultsDownloadLink`, tu préfères quelle version (same same pour moi l'un ou l'autre)

Reply:
Ah ! ma faute j'ai mal lu !
non non ça me va comme ça